packages/playground/src/components/pull-request/pull-request-conversation.tsx

```diff
@@ -0,0 +1,57 @@
+import React from 'react'
+import PullRequestSideBar from './pull-request-side-bar'
+import { EnumPullReqReviewDecision, processReviewDecision } from './utils'
+
+const mockReviewers = [
```

Comment:
move the mock data to a json file, and pass it to the component as props.
the component should not import the mock data directly.

Reply:
done

Reply:
done